docs/openapi/components/schemas/common/EntrySummary.yml

```diff
@@ -226,11 +226,19 @@ properties:
   descriptionOfMerchandise: 
     title: Description of Merchandise
     description: A description of the articles in sufficient detail to permit the classification thereof under the proper statistical reporting number in the HTS should be reported at the top of column 28. The standard definitions from the CBP HTS database are acceptable for this requirement. 
-    $ref: ./EntrySummaryLineItem.yml
+    array: 
+      items: 
+      $ref: ./EntrySummaryLineItem.yml
```

Comment:
is this indent correct?

Reply:
No it isn't. Weird that JSON Schema validator doesn't pick that up.

Reply:
Line 229 should be `type: array`. Not weird.